examples/upload.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const JWPlatformAPI = require('./jwplatform/api');
+
+const jwApiInstance = new JWPlatformAPI({
+    apiKey: process.env.API_KEY,
+    apiSecret: process.env.API_SECRET,
+});
+
+const filePath = './examples/test.mp4';
```

Comment:
Use `__dirname` to make sure you can execute script in different directores

Reply:
This example is only designed to be used in one scenario, through the Docker image (detailed in  `README`). Use of `__dirname` here breaks this functionality. Added an example to `README.md` however.